packages/manager/modules/anthos/src/dashboard/storage/remove/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,10 @@
+{
+  "anthos_dashboard_remove_storage_title": "Supprimer SVM",
+  "anthos_dashboard_remove_storage_description": "Vous supprimerez définitivement <strong>{{storageName}}</strong>. Vous perdrez tout le contenu.",
+  "anthos_dashboard_remove_storage_warning": "A La suppression entrainera l'effacement définitif de toutes les données correspondantes.",
```

Comment:
Content looks odd, was this the one provided ?

Reply:
Yeah... it was provided in specs. Probably need a review